explorer/site-config/hca-dcp/dev/mainColumn.ts

```diff
@@ -13,22 +17,17 @@ export const mainColumn = [
       {
         component: C.Section,
         props: {
-          title: "Project description",
+          collapsable: false,
+          title: "Description",
         },
         children: [
           {
-            component: C.ShowMore,
+            component: Typography,
             props: {
-              maxLines: 6,
-              buttonLabelShow: "Read More",
+              component: "p",
```

Comment:
@MillenniumFalconMechanic I'd like to review the typescript for the `Typography` props - I'd like to move the `component` prop into `explorer/site-config/hca-dcp/dev/transformer.ts`